samples/sensor/qdec/README.rst

```diff
@@ -0,0 +1,37 @@
+.. _qdec:
+
+QDEC: Quadrature Decoder
+###########################
+
+Overview
+********
+A simple quadrature decoder example
+
+Building and Running
+********************
+
+This project writes the quadrature decoder position to the console once per
```

Comment:
LGTM!
just a minor in docs `sed -i 's/once/twice/g'`

Reply:
Thanks. I forgot to update it after I changed the timeout. It should be once every 2 seconds. It's fixed.